scripts/openroad/pdn.tcl

```diff
@@ -32,7 +32,12 @@ if {[catch {read_def $::env(CURRENT_DEF)} errmsg]} {
     exit 1
 }
 
-if {[catch {pdngen $::env(PDN_CFG) -verbose} errmsg]} {
+if {[catch {convert_pdn_config $::env(PDN_CFG)} errmsg]} {
+    puts stderr $errmsg
+    exit 1
+}
```

Comment:
This command doesn't actually do anything. It's provided as a convenience to make the switch from the old to the new. The correct thing to do there is to source the `$::env(PDN_CFG)`.

Reply:
Right, I updated the change to just ```source```.